hobbyloop-api/src/main/java/hobbyloop/backend/api/infra/global/oauth2/strategy/GoogleLoadStrategy.java

```diff
@@ -0,0 +1,36 @@
+package hobbyloop.backend.api.infra.global.oauth2.strategy;
+
+import hobbyloop.backend.domain.user.SocialType;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Map;
+
+@Slf4j
+public class GoogleLoadStrategy extends SocialLoadStrategy{
```

Comment:
file format 정리부탁드립니다 
```suggestion
public class GoogleLoadStrategy extends SocialLoadStrategy {
```